### PR TITLE
Add fastTSNE recipe

### PR DESCRIPTION
**Pull request by pavlin-policar**
Add recipe for fasttsne

### CODE REVIEW

File: recipes/fasttsne/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  host:
+    - numpy >1.14
+    - pip
+    - python >=3.5
+  run:
+    - numba >=0.38.1
+    - numpy >1.14
+    - python >=3.5
+    - scikit-learn >=0.19,<0.19.99
+    - scipy
```
Comment (synapticarbors) [inline recipes/fasttsne/meta.yaml]: Do you need `pynndescent`?

File: recipes/fasttsne/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - numpy >1.14
```
Comment (synapticarbors) [inline recipes/fasttsne/meta.yaml]: See:

https://conda-forge.org/docs/meta.html#building-against-numpy

File: recipes/fasttsne/meta.yaml (hunk 1 above)
Comment (pavlin-policar) [reply recipes/fasttsne/meta.yaml]: Yes, but I couldn't find it on conda-forge, so I copied over the source over from that into a separate [branch](https://github.com/pavlin-policar/fastTSNE/tree/conda-recipe), which I'll merge once I get this working. I've also created a release 0.2.6 which comes from this branch, and the build goes through just fine.

File: recipes/fasttsne/meta.yaml (hunk 2 above)
Comment (pavlin-policar) [reply recipes/fasttsne/meta.yaml]: Could you please check if this is alright now? I have zero experience packaging conda and I really appreciate your help.

**Comment by synapticarbors** [comment]
@conda-forge/staged-recipes  @conda-forge/help-python-c 

Any idea what's going on with this `vc`, `numba`, `python` conflict that appveyor is complaining about?

```
Traceback (most recent call last):
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\environ.py", line 748, in get_install_actions
    actions = install_actions(prefix, index, specs, force=True)
  File "C:\Miniconda36-x64\lib\site-packages\conda\common\io.py", line 46, in decorated
    return f(*args, **kwds)
  File "C:\Miniconda36-x64\lib\site-packages\conda\plan.py", line 541, in install_actions
    txn = solver.solve_for_transaction(prune=prune, ignore_pinned=not pinned)
  File "C:\Miniconda36-x64\lib\site-packages\conda\core\solve.py", line 518, in solve_for_transaction
    force_remove, force_reinstall)
  File "C:\Miniconda36-x64\lib\site-packages\conda\core\solve.py", line 451, in solve_for_diff
    final_precs = self.solve_final_state(deps_modifier, prune, ignore_pinned, force_remove)
  File "C:\Miniconda36-x64\lib\site-packages\conda\core\solve.py", line 344, in solve_final_state
    solution = r.solve(tuple(final_environment_specs))  # return value is List[dist]
  File "C:\Miniconda36-x64\lib\site-packages\conda\common\io.py", line 46, in decorated
    return f(*args, **kwds)
  File "C:\Miniconda36-x64\lib\site-packages\conda\resolve.py", line 857, in solve
    self.find_conflicts(specs)
  File "C:\Miniconda36-x64\lib\site-packages\conda\resolve.py", line 244, in find_conflicts
    raise UnsatisfiableError(bad_deps)
conda.exceptions.UnsatisfiableError: The following specifications were found to be in conflict:
  - numba[version='>=0.38.1']
  - python[version='>=3.5']
  - vc=9
Use "conda info <package>" to see the dependencies for each package.
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File ".ci_support\build_all.py", line 83, in <module>
    build_all(args.recipes_dir, args.arch)
  File ".ci_support\build_all.py", line 73, in build_all
    conda_build.api.build(list(d.keys()), config=config)
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\api.py", line 201, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\build.py", line 2205, in build_tree
    notest=notest,
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\build.py", line 1343, in build
    raise e
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\build.py", line 1334, in build
    channel_urls=tuple(m.config.channel_urls))
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\environ.py", line 750, in get_install_actions
    raise DependencyNeedsBuildingError(exc, subdir=subdir)
conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform win-64: {'vc=9', "numba[version='>=0.38.1']", "python[version='>=3.5']"}
```

**Comment by synapticarbors** [comment]
@pavlin-policar -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Comment by pavlin-policar** [comment]
@synapticarbors Thanks so much for looking at this - I don't know how I would have figured this out.

**Comment by pavlin-policar** [comment]
@synapticarbors, @isuruf  Sorry to bother you guys with this again, but how long does this process typically take? I would have thought the feedstock repository would have been set up in two days?

**Comment by jakirkham** [comment]
Conversion is currently broken. ( https://github.com/conda-forge/staged-recipes/issues/6659 )

**Pull request merged**